src/main.cpp

```diff
@@ -517,10 +517,9 @@ main(int argc, char *argv[]) {
     char buffer[1024];
 
     av_log_format_line(ptr, level, fmt, vl, buffer, sizeof(buffer), &print_prefix);
-    if (level <= AV_LOG_FATAL) {
-      BOOST_LOG(fatal) << buffer;
-    }
-    else if (level <= AV_LOG_ERROR) {
+    if (level <= AV_LOG_ERROR) {
```

Comment:
```suggestion
    if (level <= AV_LOG_FATAL) {
```

Should this be "FATAL" so we still log the fatal messages, just at the lower error level?

Reply:
Nope, `AV_LOG_ERROR` is correct there because of how those log levels are defined.

```
#define AV_LOG_FATAL     8
#define AV_LOG_ERROR    16
```

So `level <= AV_LOG_ERROR` matches both `AV_LOG_FATAL` and `AV_LOG_ERROR`, but `level <= AV_LOG_FATAL` only matches `AV_LOG_FATAL`.

Reply:
Perfect!